exercises/sieve/introduction.md

```diff
@@ -0,0 +1,7 @@
+# Introduction
+
+You bought a big box of random computer parts at a garage sale.
+You've started putting the parts together to build custom computers.
+
+You want to test the performance of different combinations of parts, and decide to create your own benchmarking program to see how your computers compare.
+You choose the famous "Sieve of Eratosthenes" algorithm, an ancient algorithm, but one that should push your computers to the limits.
```

Comment:
out of curiosity, I checked whether the algorithm is indeed used as a benchmark in the wild, and my search did turn up some results, so we are all good here